python/nano/src/bigdl/nano/deps/openvino/core/model.py

```diff
@@ -82,9 +84,12 @@ def ie_network(self, model):
         if self.additional_config is not None and self._device == 'CPU':
             # TODO: check addition config based on device
             config.update(self.additional_config)
-        self._compiled_model = self._ie.compile_model(model=self.ie_network,
-                                                      device_name=self._device,
-                                                      config=config)
+        if self._device != 'VPUX' or self._precision == 'int8':
```

Comment:
what will nano do `if self._device == 'VPUX' and self._precision != 'int8'`?

Reply:
`if self._device == 'VPUX' and self._precision != 'int8'`, nano will define a `_ie_network` by `read_model`, which will be used in later `pot` process.
```
if isinstance(model, (str, Path)):
   self._ie_network = self._ie.read_model(model=str(model))
else:
   self._ie_network = model
```